src/graph.rs

```diff
@@ -354,6 +302,78 @@ impl From<String> for Value {
     }
 }
 
+impl fmt::Display for Value {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        match self {
+            Value::Null => write!(f, "#null"),
+            Value::Boolean(value) => write!(f, "{}", value),
+            Value::Integer(value) => write!(f, "{}", value),
+            Value::String(value) => write!(f, "\"{}\"", value),
```

Comment:
If you use this, it will add the quotes automatically, and also escape non-printable characters in the string:

```suggestion
            Value::String(value) => write!(f, "{:?}", value),
```

Reply:
This is the Debug formatting, right? Is that something we defined, or does it just do what we want in this case?